assets/src/scripts/monitoring-location/components/hydrograph/hydrograph-utils.js

```diff
@@ -1,3 +1,7 @@
+import cloneDeep from 'lodash/cloneDeep';
```

Comment:
I think this module (hydrograph-utils) should not be in components. How about we move it to monitoring-location/iv-data-utils?